resources/design-document.md

```diff
@@ -72,7 +72,7 @@ The front end will provide intuitive ways to interact with all of these features
 ```
 String areaId;
 String name;
-double[] location;
+List<String> location;
```

Comment:
Sorry for the unintended churn here - if this is for lat/long then keeping it as a Double is probably better than a String.